### PR TITLE
Fix ::User Profile Page - Project/Packages Tabs Have Inconsistent Layout and Style

### PR DESCRIPTION
**Pull request by puni9869**
Fix ::User Profile Page  Project Tab Have Inconsistent Layout and Style
Added the big_avator for consistency in the all header_items tabs.

https://github.com/go-gitea/gitea/issues/24871
> ### Description
> in the user profile page the `Packages` and `Projects` tab have small icons for user but other tabs have bigger profile picture with user info:
> 
> ### Screenshots
> ### **For Packages And Projects:**
> image
> 
> ### **For Other Tabs:**
> image
> 

## Before
image

 ## After changes
Project View 
image

Packages View
image

## Org view for projects page
image

## Org view for packages page
image

### CODE REVIEW

**Comment by puni9869** [comment]
@silverwind

File: routers/web/user/package.go [hunk 1]
```diff
@@ -36,7 +40,86 @@ const (
...
+		Page:     page,
+	})
+	if err != nil {
+		ctx.ServerError("GetUserFollowing", err)
+		return
+	}
+	ctx.Data["NumFollowing"] = numFollowing
+	ctx.Data["ShowUserEmail"] = setting.UI.ShowUserEmail && ctx.ContextUser.Email != "" && ctx.IsSigned && !ctx.ContextUser.KeepEmailPrivate
+	ctx.Data["EnableFeed"] = setting.Other.EnableFeed
+
```
Comment (silverwind) [inline routers/web/user/package.go:122]: Where is this code block from and can we maybe re-use a function between the two?

File: web_src/css/repo.css [hunk 2]
```diff
@@ -1917,6 +1921,11 @@
   padding-bottom: 100px;
 }
 
+.page-content .ui.ui.ui.container:not(.fluid) .empty {
```
Comment (silverwind) [inline web_src/css/repo.css:1929]: `.empty` seems an awefully generic class name. Can you be more specific, and is this prefix really needed?

**Comment by silverwind** [comment]
So this PR introduces 3 big template files, without removing the content anywhere else, so I assume this must by copy-pasted content, which we should avoid as it hurts maintainability having to do a change in many copy-pasted files (and often forgetting some places).

Instead of copying, is it possible to move only the reusable parts into `shared` folder in templates, and then importing it in 2 places? You can use the `dict` helper to pass context and if there are differences, the same helper can be used to add additional values into the dict.

**Comment by puni9869** [comment]
> 

Yes you are correct. I took the basic approach to fix this. But I like you suggestion on reusability. I will work on the reformatting and reducing the duplicate code.
💯

**Comment by puni9869** [comment]
Moved to https://github.com/go-gitea/gitea/pull/25108

**Pull request closed without merging**